### DISCUSSION TITLE
Github Copilot Shoeing no options to signin on vsCode

### DISCUSSION THREAD

**Discussion by zeeshanejaz786** [community/community#10999] [Copilot Conversations] [answered]
Today I got access to the technical preview of Github Copilot and I installed the official extension on Vs Code. I read the documentation and according to that, a window will pop up asking for authorization to the GitHub account. but nothing showed up. now I have installed multiple extensions before but they do ask for authorization. I have tried reloading the IDE and I also tried reinstalling the extension but nothing seems to have changed.
I see this message in the bottom corner of Vs Code
Copilot is encountering temperarory issues.

![image](https://user-images.githubusercontent.com/95121238/152513334-5c41d82f-46d9-4efa-8b1a-b6dc551f7ee6.png)

**Comment by zeeshanejaz786** [comment, marked as answer]
Update: issue solved automatically without me signing in manually to github copilot extension.
i guess maybe because of the github extension itself.

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
